schema/artist/index.js

```diff
@@ -44,6 +44,9 @@ const ArtistType = new GraphQLObjectType({
       id: {
         type: GraphQLString,
       },
+      follow_annotation: {
+        type: GraphQLString,
+      },
```

Comment:
To clarify: this is meaningless to any other call but the homepage artist. The schema can be enriched there.